ols-1.md

```diff
@@ -43,7 +43,7 @@ photos:
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 
-# The OLS-1 program
+# The OLS-1 Program
```

Comment:
Can you revert your changes in this file and other `ols-x.md`?

Reply:
im not sure how to go about that

Reply:
You can make the changes (ex. put the hyphen and the old titles back) and commit these changes, @1Joda